bobtimus/src/http.rs

```diff
@@ -13,7 +21,8 @@ where
     R: RngCore + CryptoRng + Clone + Send + Sync + 'static,
     RS: LatestRate + Clone + Send + Sync + 'static,
 {
-    let health = warp::get().and(warp::path::end()).map(warp::reply);
```

Comment:
Are we dumping the content of index.html in the terminal now in waves script? Would be nice if not :)

Reply:
Fixed in the last commit ;)